OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -99,19 +98,16 @@ def main():
         hutil = parse_context(operation)
         exit_code = operations[operation](hutil)
 
-        # For common problems, provide a more descriptive message
+        # Exit code 1 indicates a general problem that doesn't have a more
+        # specific error code; it often indicates a missing dependency
         if exit_code is 1 and operation == 'Install':
             message = 'Install failed with exit code 1. Please check that ' \
                       'dependencies such as curl and python-ctypes are ' \
                       'installed.'
```

Comment:
Can you remove dependency package names from the message? Put something like "check dependencies".
Also can you extend this message to say "for details check extension log"?